cmd/zc_traverser_s3.go

```diff
@@ -213,15 +211,18 @@ func newS3Traverser(rawURL *url.URL, ctx context.Context, recursive, getProperti
 	return
 }
 
-// Disallow the non-region aware URL type.
+// Discourage the non-region aware URL type. (but don't ban it, because that breaks almost all our S3 automated tests)
```

Comment:
_phwoof_

Hopefully we can figure out a better workaround for regionless S3 URLs in the future.